File: Javascript/A_NivelIniciante/Exercicio_2807.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var qtd = parseInt(lines.shift());
var iccanobif = [1,1];
for(let i =1;i<qtd-1;i++){
    iccanobif.push((iccanobif[i] + iccanobif[i-1]));
}

if(qtd!=1 || qtd!=1){
    console.log(iccanobif.reverse().join(" "));
}else{
    console.log(1)
}

File: Python/A_NivelIniciante/Exercicio_2807.py
# -*- coding: cp1252 -*-
# -*- encoding: utf-8 -*-
qtd = int(input())
iccanobif = [1,1]
for x in range(1,qtd-1):
    iccanobif.append((iccanobif[x] + iccanobif[x-1]))
iccanobif.reverse()
if qtd!=1 or qtd!=1:
    print(' '.join(map(str, iccanobif)))
else:
    print(1)


